magma/home/__init__.py

```diff
@@ -21,14 +21,26 @@
 @home_page.route('/', methods=['GET', 'POST'])
```

Comment:
I'd like to split this into two routes. One for `@home_page.route('/', methods=['GET']) ` and one for `@home_page.route('/', methods=['POST']) ` but thought reviewing initially like this would be easier.

I think this would make the code easier to follow. Thoughts?